src/generic/UnivPoly.jl

```diff
@@ -33,7 +33,7 @@ parent_type(::Type{UnivPoly{T, U}}) where {T <: RingElement, U <: AbstractAlgebr
    UniversalPolyRing{T, U}
 
 function mpoly_ring(S::UniversalPolyRing{T, U}) where {T <: RingElement, U <: AbstractAlgebra.MPolyRingElem{T}}
-   return S.mpoly_ring::Generic.MPolyRing{T}
+   return S.mpoly_ring::MPolyRing{T}
```

Comment:
The one thing I like about the old version is that it is clear that this the concrete type, not the abstract one with the same name :-/

Reply:
Yeah that's true. But since it wasn't used consistently throughout the `Generic` module, I don't know how much they actually help. If you want to keep these, I can remove all removals of `Generic.` like this from this PR.